docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
This new link isn't actually right, and I don't know if the original reference is still out there at all.

The ipython/nbconvert repo this referred to got merged into IPython several releases before jupyter/nbconvert was split back out. So jupyter/nbconvert issue 137 is not ipython/nbconvert 137.